test/custom-cache.test.js

```diff
@@ -29,7 +29,8 @@ Object.keys(testAPIs).forEach(API => {
       rmrf.sync(dbPath)
       ipfsd = await startIpfs(API, config.daemon1)
       ipfs = ipfsd.api
-      orbitdb1 = new OrbitDB(ipfs, dbPath + '/1', {
+      orbitdb1 = await OrbitDB.createInstance(ipfs, {
+        directory: dbPath + '/1', 
```

Comment:
We should prolly use `path.join(dbPath, '1')` here to make sure paths are joined correctly (also on Windows).